src/v/cluster/topics_frontend.h

```diff
@@ -189,6 +190,10 @@ class topics_frontend {
     ss::future<result<std::vector<partition_state>>>
       get_partition_state(model::ntp);
 
+    /// Prints a warning log for every topic that has an RF less than the
+    /// minimum specified in minimum_topic_replication_factor
```

Comment:
nit: spell out what `RF` means for newcomers, specify that minimum_topic_replication_factor is a cluster config